infra/core/networking/private-endpoint.bicep

```diff
@@ -39,22 +39,21 @@ resource privateEndpoint 'Microsoft.Network/privateEndpoints@2021-02-01' = {
       }
     ]
   }
-}
-
-resource privateDnsZoneGroup 'Microsoft.Network/privateEndpoints/privateDnsZoneGroups@2020-06-01' = {
-  parent: privateEndpoint
-  name: '${name}-group'
-  properties: {
-    privateDnsZoneConfigs: [
-      {
-        name: 'config1'
-        properties: {
-          privateDnsZoneId: dnsZoneId
+  resource privateDnsZoneGroup 'privateDnsZoneGroups' = {
```

Comment:
Make sure bicep formatting is run on all files.  There should be a line break b/w resources